sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -923,7 +923,8 @@ glsTextureTestUtil.computeLodFromDerivates3D = function(mode, dudx, dvdx, dwdx,
             DE_ASSERT(false);
     }
 
-    return Math.log(p) * 1.44269504089;
+    // Native dEQP uses 32-bit numbers. So here 64-bit floating numbers should be transformed into 32-bit ones to ensure the correctness of the result.
+    return new Float32Array([Math.log(p)])[0] * new Float32Array([1.44269504089])[0];
```

Comment:
JS number has higher precision than float in C++. This may cause different comparison result from C++.  Jiawei can explain it further about the case in this test.

New an array each time hurts performance. Thanks for suggestion. It should be fixed.